### PR TITLE
DB-9647: identify duplicate region correctly

### PR DESCRIPTION
**Pull request by jyuanca**

### CODE REVIEW

File: platform_it/src/test/java/com/splicemachine/test/SpliceTestPlatformConfig.java [hunk 1]
```diff
@@ -247,7 +247,7 @@ public static Configuration create(String hbaseRootDirUri,
         //
         // Memstore, store files, splits
         //
-        config.setLong(HConstants.HREGION_MAX_FILESIZE, 32 * MiB); // hbase.hregion.max.filesize
+        config.setLong(HConstants.HREGION_MAX_FILESIZE, 320 * MiB); // hbase.hregion.max.filesize
```
Comment (dgomezferro) [inline platform_it/src/test/java/com/splicemachine/test/SpliceTestPlatformConfig.java:250]: Why are we increasing this parameter?

**Comment by spliceci** [comment]
TEST FAILED

**Comment by spliceci** [comment]
TEST FAILED

**Comment by jyuanca** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST FAILED

**Comment by jyuanca** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

**Pull request merged**